iphone/cli/hooks/frameworks.js

```diff
@@ -57,7 +64,12 @@ class FrameworkManager {
 			post: (builder, callback) => {
 				this._logger.trace('Starting third-party framework detection');
 				this._builder = builder;
-				this.detectFrameworks().then(callback, callback);
+				this.detectFrameworks().then(callback, e => {
+					if (this._cli.argv.platform === 'ios') {
+						this._logger.error(e.stack);
+					}
+					callback(e);
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/cli/hooks/...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/frameworks.js;
  Line: 71;
-->

- :warning: iphone/cli/hooks/frameworks.js line 71 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)